dojo/context_processors.py

```diff
@@ -16,7 +16,8 @@ def globalize_oauth_vars(request):
             'SOCIAL_AUTH_KEYCLOAK_LOGIN_BUTTON_TEXT': settings.SOCIAL_AUTH_KEYCLOAK_LOGIN_BUTTON_TEXT,
             'SAML2_ENABLED': settings.SAML2_ENABLED,
             'SAML2_LOGIN_BUTTON_TEXT': settings.SAML2_LOGIN_BUTTON_TEXT,
-            'SAML2_LOGOUT_URL': settings.SAML2_LOGOUT_URL}
+            'SAML2_LOGOUT_URL': settings.SAML2_LOGOUT_URL,
+            'DOCUMENTATION_URL': settings.DOCUMENTATION_URL}
```

Comment:
I am bit surprised to see the URL here. What is the reason for that?

Reply:
If was the easiest way how to propagate this setting to all pages. What do you recommend?

Reply:
Or I can put it in to `system_settings`. Maybe it is the best idea.

Reply:
To me `settings.dist.py` is the right place for a configuration like this. Just finding out how to use it in a template.

Reply:
I already changed it to `system_settings`. I prefer settings that can be changed on running instances.